content/blog/thank-you-reviewers-2020.md

```diff
@@ -0,0 +1,171 @@
+[Twitter](https://www.twitter.com/greatdevaks)
+[Linkedin](https://www.linkedin.com/in/greatdevaks)
+
+
+### Archana Iyer
+    Archana is an AI Engineer at Continental Automotive Singapore. She is a graduate student, based out of Singapore. She worked as a Deep Learning Engineer at a Pharma IT Company for 1.5 years where I worked on deep learning and NLP Applications. Apart from that, she has a keen interest in edge computing and computer vision. She created a course, “Choosing the Right Hardware for the Edge” for the Intel AI on Edge for IoT Developers Nanodegree on Udacity in April 2020. In her free time, she works at Women Who Code as a Leadership fellow for the Python/Cloud Tracks.
+
+
+### Charmi Chokshi
+    Charmi is a Machine Learning Engineer and one of the youngest Google Developers Expert in ML. She loves helping developers & students looking forward to growing their careers in the field of Artificial Intelligence via tech talks, mentoring sessions, and blogs.
+[Twitter](https://twitter.com/CharmiChokshi)
+
+
+### Chetan Giridhar
+    Chetan is a Python enthusiast and open source evangelist. He has been a speaker at Python conferences and takes keen interest in platform engineering, distributed systems and real time cloud communications. In his spare time Chetan reads about the world history and loves helping startups.
+
+
+### Derrick Joseph
+     Derrick is former PyCon speaker and has been a Pythonista since 2015, he started his career as a Data Scientist and later moved to Engineering. He has experience working with ML, Systems and has implemented business processes automations as generic frameworks.
+His current area of exploration is CPython and is working with Linkedin's Production Infrastructure Engineering team.
+[Twitter](https://twitter.com/C0d3eswaran)
+[Linkedin]( https://www.linkedin.com/in/derrickjosephknatt)
+
+
+### Dhruva Shashtri 
+    Her interests as an inquisitive #WomenInTech is to learn new every day so she continually looks out for areas that interest her to work on. Being a facilitator for #IamRemarkable initiative and community passionate, playing a role as an Ambassador at WTM Ahmedabad, enthusiastic about emboldening women in every sphere of their life and enjoys people’s synergy.
+[Twitter](https://twitter.com/dhruvashastri12)
+
+
+### Girish Joshi
+    Girish  is a polyglot programmer and a free and open source enthusiast from Pune. He has been a part of the Python Pune community since 2015.
```

Comment:
```suggestion
    Girish is a polyglot programmer and a free and open source enthusiast from Pune. He has been a part of the Python Pune community since 2015.
```